playbooks/roles/ambari-server/tasks/main.yml

```diff
@@ -22,6 +22,20 @@
     state: latest
   notify: Reload systemd
 
+- name: Create missing ambari-server symlinks
```

Comment:
This may be a version issue, I am not seeing this with the current install

Reply:
This fixed the ImportError: No module named ambari_commons.exceptions.

Reply:
This was the fix for the ImportError: No module named ambari_commons.exceptions that the support agent found.